tests/vectors_kem.c

```diff
@@ -11,15 +11,34 @@
 #include <sys/stat.h>
 
 #include <oqs/oqs.h>
-
+#include <oqs/sha3.h>
 #include "system_info.c"
 
+#ifdef OQS_ENABLE_KEM_ML_KEM
+/* macros for sanity checks for encaps and decaps key */
+#define ML_KEM_BLOCKSIZE        384
+#define ML_KEM_K_MAX            4
+#define ML_KEM_N                256
+#define ML_KEM_1024_PK_SIZE     1568
```

Comment:
```suggestion
```

This can be replaced with the existing `OQS_KEM_ml_kem_1024_length_public_key` macro.

Reply:
The macro 'OQS_KEM_ml_kem_1024_length_public_key' is only generated if 'ML-KEM-1024' is enabled. incase of builds(minimal builds?) with 'ML-KEM-1024' not enabled will result in the macro not being generated and compilation error henceforth. so, decide to hardcode here.

Reply:
Good point!